x-pack/legacy/plugins/lens/public/plugin.tsx

```diff
@@ -38,6 +38,8 @@ import {
 import { FormatFactory } from './legacy_imports';
 import { IEmbeddableSetup, IEmbeddableStart } from '../../../../../src/plugins/embeddable/public';
 import { EditorFrameStart } from './types';
+import { setup as visualizations } from '../../../../../src/legacy/core_plugins/visualizations/public/np_ready/public/legacy';
```

Comment:
Please move this import to `legacy.ts` and pass it in via the `setup` call to keep the shim clean.